deployments/deploy-ssh.sh

```diff
@@ -547,6 +551,7 @@ spec:
             "--pure-volumes=$IS_PURE_VOLUMES",
             "--pure-fa-snapshot-restore-to-many-test=$PURE_FA_CLONE_MANY_TEST",
             "--pure-san-type=$PURE_SAN_TYPE",
+            "--pure-fada-pod=$PURE_FADA_POD",
```

Comment:
@Pure-AdamuKaapan one quick question, will there be a scenario where we need to test multiple apps with different pod names?

Reply:
Yes but we aren't tackling that yet, we can figure out the specifics later (as noted in the comment from Vishal)